### ISSUE TITLE
companion bugs default https host for nginx proxy

### ISSUE THREAD

**Issue by ipepe** [JrCs/docker-letsencrypt-nginx-proxy-companion#373]
Hi. I noticed that when using companion (separate containers scenario) there are no `default.key` and `default.crt` and that makes `nginx.tmpl` to not generate a default (and first) 'server' in configuration that would respond with 503. Because of that, when creating such stack, anyone can query my server on https by ip and nginx will respond with first valid https configuration, and not 503 like it should. How can we fix this?

**Comment by buchdag** [comment]
Hi.

Correct me if I'm wrong, but I think the same thing would happen even if one does not use `letsencrypt-nginx-proxy-companion` at all and just manually put certificates on the `/etc/nginx/certs` folder as described in the `nginx-proxy` doc.

**Comment by ipepe** [comment]
Yeah, I think You are right. Today I figured out how to setup this correctly, although it would be nice to have this information in readme or somewhere.

So basically I change my claim on this issue from "Bug" to "Needs more information in Readme"

**Comment by vicary** [comment]
I guess my issue #411 is describing the same thing, but I am not sure if we should simply document this and call it a day.

Making up some invalid placeholder certs for default doesn't quite make sense in production.

I see that users of `nginx-proxy` is very likely to start without a valid default upstream, and may never will.

How about disabling access to default server in the default template for `nginx-proxy` and rely solely on `VIRTUAL_HOST`? Users do need a default upstream can always override it with a separate `docker-gen` container.
